.buildkite/scripts/build-storybook.sh

```diff
@@ -5,7 +5,7 @@ set -e
 . ".buildkite/scripts/helpers/setup-registry.sh"
 
 yarn install --frozen-lockfile
-yarn workspace @kaizen/design-tokens prepublish
-yarn workspace @kaizen/tailwind prepublish
+yarn workspace @kaizen/design-tokens publish
```

Comment:
I think `publish` runs extra stuff?
If this is similar to other package commands, `prepublish` _may_ have been replaced with `version`.

But if you've checked that it doesn't have any side-effects, then it will be fine :)

Reply:
Missed this, I toyed with `publish` but just went back to `prepublish`